backend/protocol_rpc/endpoints.py

```
@@ -501,10 +506,285 @@ def get_transaction_count(
+        and transaction_data["consensus_data"]["leader_receipt"] is not None
+        and "pending_transactions"
+        in transaction_data["consensus_data"]["leader_receipt"]
+        and transaction_data["consensus_data"]["leader_receipt"]["pending_transactions"]
+        is not None
+    ):
+        for message in transaction_data["consensus_data"]["leader_receipt"][
+            "pending_transactions"
+        ]:
+            pending_transactions.append(
+                [
+                    message["address"] if "address" in message else "",  # Account
```

Ruff (0.11.9)

Use `message.get("address", "")` instead of an `if` block

Replace with `message.get("address", "")`

(SIM401)